action.yml

```
@@ -25,8 +25,17 @@ runs:
       run: |
         julia -e '
           using JuliaFormatter
-          format(".") ? exit(0) : exit(1)'
+          format(".")'
       shell: bash
+    - name: Check for formatting errors
+      shell: bash
+      run: |
+        output=$(git diff --name-only)
+        if [ "$output" != "" ]; then
+            echo 1>&2 "Some files have not been formatted !!!"
```

Just a style suggestion:

```suggestion
            >&2 echo "Some files have not been formatted !!!"
```